relay-dynamic-config/src/defaults.rs

```diff
@@ -25,12 +28,71 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
         return;
     }
 
+    // Add conditions to filter spans if a specific module is enabled.
+    // By default, this will extract all spans.
+    let span_op_field_name = "span.op";
+    let span_op_conditions = if project_config
+        .features
+        .has(Feature::SpanMetricsExtractionAllModules)
+    {
+        None
+    } else {
+        let mut rules: Vec<RuleCondition> = Vec::new();
+
+        if project_config
+            .features
+            .has(Feature::SpanMetricsExtractionDBModule)
+        {
+            rules.push(RuleCondition::And(AndCondition {
+                inner: vec![
+                    RuleCondition::Glob(GlobCondition {
+                        name: span_op_field_name.into(),
+                        value: GlobPatterns::new(vec!["db*".into()]),
+                    }),
+                    RuleCondition::Not(NotCondition {
+                        inner: Box::new(RuleCondition::Eq(EqCondition {
+                            name: span_op_field_name.into(),
+                            value: Value::Array(vec![
+                                "db.clickhouse".into(),
+                                "db.redis".into(),
+                                "db.sql.query".into(),
```

Comment:
@AbhiPrasad what do you think about `"db.sql.query"`? I feel like we should allow this by default, and opt out the one problematic project that came up so far

Reply:
ah yeah I think we should allow it - good catch!